pkg/skuba/actions/node/remove/remove_test.go

```diff
@@ -136,9 +137,24 @@ apiServer:
 		tt := tt // Parallel testing
 		t.Run(tt.name, func(t *testing.T) {
 			tt.clientset.CoreV1().ConfigMaps(metav1.NamespaceSystem).Create(cm)
+
+			shaTarget := fmt.Sprintf("%x", sha1.Sum([]byte(tt.target)))
+			shaExecuter := ""
+			for _, executer := range tt.executer {
+				shaExecuter = fmt.Sprintf("%x", sha1.Sum([]byte(executer)))
+				tt.clientset.BatchV1().Jobs(metav1.NamespaceSystem).Create(&batchv1.Job{
+					ObjectMeta: metav1.ObjectMeta{
+						Name:      fmt.Sprintf("caasp-remove-etcd-member-%.10s-from-%.10s", shaTarget, shaExecuter),
+						Namespace: metav1.NamespaceSystem,
+					},
+					Spec:   jobSpec,
+					Status: batchv1.JobStatus{Active: 1},
+				})
```

Comment:
Job mocking here is a indirect way to bypass jobs processing logic and job verification when calling `kubernetes.CreateAndWaitForJob`. 

For now, it is OK at least to make tests not blocking to wait job finished.